src/transformers/models/mask2former/image_processing_mask2former.py

```diff
@@ -772,7 +772,7 @@ def preprocess(
             ignore_index,
             do_reduce_labels,
             return_tensors,
-            input_data_format=input_data_format,
+            input_data_format=data_format,
```

Comment:
For this model (and the ones below with this change) the "input_data_format" is now data_format because the images were changed to data_format in the [_preprocess_image](https://github.com/huggingface/transformers/blob/0ed3ffcb4461a244b87781a24e5ebd0a78f98142/src/transformers/models/mask2former/image_processing_mask2former.py#L638) step.

This wasn't previously caught because the test images and the defaults were always in channels_last format.

Now, we're feeding in channels_first numpy images for tests, as this is the standard format for numpy images.